### DISCUSSION TITLE
Missing the "Argus" models

### DISCUSSION THREAD

**Discussion by superdioz** [AT0myks/reolink-fw-archive#59] [Missing firmware]
Specifically I was looking for the "Reolink Argus Eco" model, I have two hw versions of them:
1. BIPC_561MIX16M82B1663MP at v3.0.0.1649_22122701
2. BIPC_36S3216MGUN3 at 0616_722_52_481

but there aren't also the rest of the Argus (2, 2e, pro, etc.).
For some reasons (as usual with reolink) they aren't in the donwload center anymore.

ps. Thanks a lot for the work you are doing!

**Comment by Dnominated** [comment]
I believe you have to contact Reolink and ask them to push a new firmware image. That is a battery-operated camera, and for some reason their firmware images are set-up that way.

**Comment by AT0myks** [comment]
Unfortunately I've never seen a firmware file for any of the battery-powered cameras, and I don't think they've ever been available on the download center or even in the wild. I'd like to see one some day, and when that happens it'll be added here.
